tests/openapi/runner/test_runner.py

```diff
@@ -92,27 +92,26 @@ def test_pre_validate_openapi_file(self) -> None:
         result = runner.pre_validate_file(file_content)
         self.assertTrue(result)
 
-    # def test_runner_results_consistency(self) -> None:
-    #     current_dir = os.path.dirname(__file__)
-    #     valid_dir_path = os.path.join(current_dir, "resources")
-    #     results_file_path = os.path.join(current_dir, "resources/runner_results/results.sarif")
-    #     runner = Runner()
-    #     checks = ["CKV_OPENAPI_1", "CKV_OPENAPI_4", "CKV_OPENAPI_3"]
-    #     report = runner.run(
-    #         root_folder=valid_dir_path,
-    #         runner_filter=RunnerFilter(framework='openapi', checks=checks)
-    #     )
-    #     self.assertEqual(len(report.failed_checks), 12)
-    #     self.assertEqual(report.parsing_errors, [])
-    #     self.assertEqual(len(report.passed_checks), 6)
-    #     self.assertEqual(report.skipped_checks, [])
-    #
-    #     with open(results_file_path) as f:
-    #         expected_report_dict = json.loads(f.read())
-    #     json_sarif_report = report.get_sarif_json("test")
-    #     del json_sarif_report["runs"][0]["tool"]
-    #     self.assertEqual(json_sarif_report, expected_report_dict)
+    def test_runner_results_consistency(self) -> None:
+        current_dir = os.path.dirname(__file__)
+        valid_dir_path = os.path.join(current_dir, "resources")
+        results_file_path = os.path.join(current_dir, "resources/runner_results/results.sarif")
+        runner = Runner()
+        checks = ["CKV_OPENAPI_1", "CKV_OPENAPI_4", "CKV_OPENAPI_3"]
+        report = runner.run(
+            root_folder=valid_dir_path,
+            runner_filter=RunnerFilter(framework='openapi', checks=checks)
+        )
+        self.assertEqual(len(report.failed_checks), 12)
+        self.assertEqual(report.parsing_errors, [])
+        self.assertEqual(len(report.passed_checks), 6)
+        self.assertEqual(report.skipped_checks, [])
+
+        with open(results_file_path) as f:
+            expected_report_dict = json.loads(f.read())
+        json_sarif_report = report.get_sarif_json("test")
+        self.assertEqual(len(json_sarif_report["runs"][0]["results"]), len(expected_report_dict["runs"][0]["results"]))
```

Comment:
shouldn't we assert also the content?

Reply:
It should be enough in this test's context